doc/whats-new.rst

```diff
@@ -82,6 +82,8 @@ Documentation
   By `Zach Moon <https://github.com/zmoon>`_.
 - Raise a more informative error when trying to open a non-existent zarr store. (:issue:`6484`, :pull:`7060`)
   By `Sam Levang <https://github.com/slevang>`_.
+- Added examples to docstrings for DataArray.expand_dims(), DataArray.drop_duplicates(), DataArray.reset_coords(), DataArray.equals(), DataArray.identical(), DataArray.broadcast_equals(), DataArray.bfill(), DataArray.ffill(), DataArray.fillna(), DataArray.dropna(), DataArray.drop_isel(), DataArray.drop_sel(), DataArray.head(), DataArray.tail(). (:issue:`5816`, :pull:``).
```

Comment:
Using `:py:meth:` in restructured text documents will create a hyperlink to the relevant method.

```suggestion
- Added examples to docstrings for :py:meth:`DataArray.expand_dims()`, :py:meth:`DataArray.drop_duplicates()`, :py:meth:`DataArray.reset_coords()`, :py:meth:`DataArray.equals()`, :py:meth:`DataArray.identical()`, :py:meth:`DataArray.broadcast_equals()`, :py:meth:`DataArray.bfill()`, :py:meth:`DataArray.ffill()`, :py:meth:`DataArray.fillna()`, :py:meth:`DataArray.dropna()`, :py:meth:`DataArray.drop_isel()`, :py:meth:`DataArray.drop_sel()`, :py:meth:`DataArray.head()`, :py:meth:`DataArray.tail()`. (:issue:`5816`, :pull:`7088`).
```

Reply:
Cool, thanks!